packages/alfa-dom/src/h.ts

```diff
@@ -1,29 +1,31 @@
-import { Option, None } from "@siteimprove/alfa-option";
+import {None, Option} from "@siteimprove/alfa-option";
 
-import { Namespace } from "./namespace";
+import {Namespace} from "./namespace";
 
-import { Node } from "./node";
-import { Attribute } from "./node/attribute";
-import { Document } from "./node/document";
-import { Element } from "./node/element";
-import { Fragment } from "./node/fragment";
-import { Text } from "./node/text";
-import { Type } from "./node/type";
+import {Node} from "./node";
+import {Attribute} from "./node/attribute";
+import {Document} from "./node/document";
+import {Element} from "./node/element";
+import {Fragment} from "./node/fragment";
+import {Text} from "./node/text";
+import {Type} from "./node/type";
 
-import { Block } from "./style/block";
-import { Declaration } from "./style/declaration";
+import {Block} from "./style/block";
+import {Declaration} from "./style/declaration";
 import {
-  Rule,
   FontFaceRule,
   KeyframeRule,
   KeyframesRule,
   MediaRule,
   NamespaceRule,
   PageRule,
+  Rule,
   StyleRule,
   SupportsRule,
 } from "./style/rule";
-import { Sheet } from "./style/sheet";
+import {Sheet} from "./style/sheet";
+import {Shadow} from "./node/shadow";
+import {Predicate} from "@siteimprove/alfa-predicate";
```

Comment:
Looks like something overruled Prettier?

Reply:
🤯 
I am experiencing problems with my Prettier setup not working since the latest IDE upgrade 😞